app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/FahrplanFragment.java

```diff
@@ -289,6 +290,21 @@ private void viewDay(boolean reload) {
         updateNavigationMenuSelection();
     }
 
+    private Map<Integer, List<Lecture>> groupLecturesPerRoom(List<Lecture> lectures) {
+        Map<Integer, List<Lecture>> roomLectures = new HashMap<>();
+        for (int roomIndex = 0; roomIndex < MyApp.roomCount; roomIndex++) {
+            for (Lecture lecture : lectures) {
+                if (lecture.roomIndex == roomIndex) {
+                    if (!roomLectures.containsKey(roomIndex)) {
+                        roomLectures.put(roomIndex, new ArrayList<>());
+                    }
+                    roomLectures.get(roomIndex).add(lecture);
```

Comment:
:mag: Please suppress Lint NPE warning here.

Reply:
Cant suppress this warning I guess.

Reply:
outdated